src/commands/retrieve.js

```diff
@@ -0,0 +1,24 @@
+const Discord = require('discord.js');
+const logger = require('./../logger');
+const db = require('./../mongo');
+
+module.exports = {
+	retrieve : {
+		name : 'retrieve',
+		args : true,
+		description : 'Retrieve a command from the database.',
+		async execute(message, data) {
+			const mongo = await db;
```

Comment:
instead of calling this in each command, call it just once in the bot.js and then pass it as a parameter to your `execute`